### DISCUSSION TITLE
How to increase token limit?

### DISCUSSION THREAD

**Discussion by ltsimple** [imartinez/privateGPT#1752] [Q&A] [answered]
When I try to use the model with a rather big csv file. I get a Value Error about the token limit.
ValueError: Initial token count exceeds the token limit

I think I need to increase it but I couldn't figure out how to do it.

**Comment by luchontandil** [comment]
Same issue here, im trying to ask stuff from a 40mb csv file, Even saying "Hi" returns the same error

**Comment by dbzoo** [comment, marked as answer]
See my suggestion here; https://github.com/imartinez/privateGPT/issues/1701
